app/src/main/java/com/courseproject/tindar/controllers/likelist/LikeListController.java

```diff
@@ -0,0 +1,18 @@
+package com.courseproject.tindar.controllers.likelist;
+
+import com.courseproject.tindar.usecases.likelist.LikeListInputBoundary;
+
+public class LikeListController {
```

Comment:
Class javadoc

Reply:
Honestly every confused about why this exists, because the only thing this class currently does is hold a LikeListInputBoundary object and call that object's methods.

Reply:
Controller connects UI/front end to backend

Reply:
Can you elaborate on this and put it in the javadoc? Because right now it looks like this connects front end to a LikeListInputBoundary, and a LikeListInputBoundary connects to a LikeListDsGateway, and the LikeListDsGateway connects to the actual database.

Reply:
Can you elaborate on this and put it in the javadoc? Because right now it looks like this connects front end to a LikeListInputBoundary, and a LikeListInputBoundary connects to a LikeListDsGateway, and the LikeListDsGateway connects to the actual database.